### DISCUSSION TITLE
Is it possible to activate dislike counts in YouTube shorts?

### DISCUSSION THREAD

**Discussion by DeFaLT-dj** [Anarios/return-youtube-dislike#440] [Ideas]
If you don't know YouTube shorts for YouTube web on desktop got a dedicated UI and since YouTubes hides dislikes, this one hasn't either. I wondered if it is possible to make dislikes available on shorts. Because unlike normal YouTube videos, shorts never showed dislike counts. If it is somehow possible it would be cool to see it in future versions of return YouTube dislikes :)

![image](https://user-images.githubusercontent.com/64617514/149287893-e612fd10-09f6-490c-a911-5e1bb9ba676e.png)

**Comment by sy-b** [comment]
yes

**Reply by sy-b** [reply]
Available in beta

**Comment by PickleNik** [comment]
https://chrome.google.com/webstore/detail/return-youtube-dislike/ccdpldgbfmaenfmolngblkjlfpheogpa?hl=en&authuser=0
(the beta link)

**Comment by Anarios** [comment]
Desktop Shorts support is available in [v4.0.5](https://github.com/Anarios/return-youtube-dislike/releases/tag/v4.0.5), including fixes for missing dislike counts, newer Shorts layouts, and scrolling between Shorts. Closing this feature request as resolved. Please update and reload YouTube.

The Firefox store update is still awaiting Mozilla review.
